tests/data/group_system/group_accounts/group_accounts-restrictions/group_password_expiration/rule_accounts_password_minlen_login_defs/password_minlen_15.pass.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+#
+# profiles = xccdf_org.ssgproject.content_profile_ospp
+# platform = multi_platform_fedora,Red Hat Enterprise Linux 8
+
+if grep -q "^PASS_MIN_LEN" /etc/login.defs; then
+	sed -i "s/^PASS_MIN_LEN.*/PASS_MIN_LEN 15/" /etc/login.defs
```

Comment:
here the value changed from 12 -> 15. since the ospp profile is not setting the variable for this rule, it uses the default (15)

Reply:
It should be 12 for RHEL8 because RHEL8 OSPP will set 12. See that we set in #4405 var_accounts_password_minlen_login_defs=12

Reply:
that means that filename `password_minlen_12.fail.sh` should be reverted to `password_minlen_12.pass.sh`. But test will fail until new RHEL8/OSPP profile gets in place

Reply:
what about keeping `password_minlen_12.pass.sh` test only for fedora standard profile until OSPP gets in place?

Reply:
The problem here is that even OSPP (which is the one selected in the `profiles` test scenario header) from fedora doesn't select the variable value explicit, so it is using the default value which is `15` and the test will fail aswell. Not sure what to do here.

Reply:
I meant `standard` profile for Fedora, but yea it's hard to say what to do with this one in OSPP. I don't know if failing test scenario is ok for few days.
@jan-cerny do you have any suggestion?

Reply:
In general, we can have different test scenarios for different profiles and different platforms. Eg. first scenario for Fedora standard profile that uses value 15 and second scenario for RHEL8 OSPP profile that uses value 12.

Regarding the fact that the tests are currently failing. The most correct solution would be to hold this PR until the RHEL 8 OSPP profile is aligned with the CC requirements and then rebase on top of the master.
However, the tests are not a part of PR gating, so we can say that we have prepared the test scenarios for future and they will start to pass once the OSPP profile is fixed (seems almost like test driven development).